cmd/podman/restore.go

```diff
@@ -79,5 +79,14 @@ func restoreCmd(c *cliconfig.RestoreValues, cmd *cobra.Command) error {
 	if (c.Import != "") && (c.All || c.Latest) {
 		return errors.Errorf("Cannot use --import and --all or --latest at the same time")
 	}
+
+	argLen := len(c.InputArgs)
+	if (c.All || c.Latest) && argLen > 0 {
+		return errors.Errorf("no arguments are needed with --all or --latest")
+	}
+	if argLen < 1 && !c.All && !c.Latest && (c.Import == "") {
```

Comment:
Parenthesis are not required here.

Reply:
Confirmed, fixed, tested. I will force-push the update once I hear back re: `--import`. Thank you.